client/src/components/CommentApp/components/CommentReply/index.tsx

```diff
@@ -215,8 +215,8 @@ export default class CommentReplyComponent extends React.Component<CommentReplyP
     return (
       <>
         <CommentHeader commentReply={reply} store={store} focused={isFocused} />
-        <p className="comment-reply__text">{reply.text}</p>
-        <div className="comment-reply__progress">{gettext('Deleting')}</div>
+        <p className="comment__text">{reply.text}</p>
+        <div className="comment__progress">{gettext('Deleting')}</div>
       </>
```

Comment:
I couldn’t figure out how to get this state but decided to move on in the interest of time.